db/version_set.cc

```diff
@@ -475,9 +475,10 @@ bool Version::OverlapInLevel(int level,
 
 int Version::PickLevelForMemTableOutput(
     const Slice& smallest_user_key,
-    const Slice& largest_user_key) {
+    const Slice& largest_user_key,
+    const int level_limit) {
   int level = 0;
-#if 0
+#if 1
```

Comment:
Since this code is being put back into production, should remove the "#if 1/#endif" conditional compilation statements.